lib/opensusebasetest.pm

```diff
@@ -583,6 +566,26 @@ sub wait_boot {
     $self->{in_wait_boot} = 0;
 }
 
+sub wait_boot_on_local_disk {
+   # assuming the cursor is on 'installation' by default and 'boot from
+   # harddisk' is above
+   send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
+   boot_local_disk;
+   my @tags = qw(grub2 tianocore-mainmenu);
+   push @tags, 'encrypted-disk-password-prompt' if (get_var('ENCRYPT'));
+
+   check_screen(\@tags, 15)
+              || die 'neither grub2 nor tianocore-mainmenu needles found';
```

Comment:
Please replace that by just `assert_screen`

Reply:
Done.